lib/config/presets/github/index.spec.ts

```diff
@@ -98,6 +98,20 @@ describe('config/presets/github/index', () => {
       expect(content).toEqual({ foo: 'bar' });
     });
 
+    it('should query preset within the file when .json5 extension provided', async () => {
+      httpMock
+        .scope(githubApiHost)
+        .get(`${basePath}/somefile.json5`)
+        .reply(200, {
+          content: toBase64('{"foo":"bar"}'),
+        });
+      const content = await github.getPreset({
+        repo: 'some/repo',
+        presetName: 'somefile.json5',
+      });
+      expect(content).toEqual({ foo: 'bar' });
+    });
+
```

Comment:
It's a change in `presets/utils.ts` but the existing tests are mocking the response of the `fetch` so I think it gives more value in `presets/github/index.spec.ts`

If you agree I can replicate the same tests for the other `presetSources`.

Reply:
Already pushed for other sources. Let me know if it's ok and I can refactor it if necessary.